Picturepark.ContentPortal.Demo/Picturepark.ContentPortal.Demo/ClientApp/src/app/utilities/translations.ts

```diff
@@ -0,0 +1,105 @@
+    Title: {
+      en: 'Welcome to the FOFA Content Portal!',
+      de: 'Willkommen im FOFA Content Portal!',
+    },
+    DemoInfo: {
+      en: 'More info about this demo',
+      de: 'Weitere infos zu dieser demo',
+    },
+    Content: {
+      en: `Our association is all about sharing knowledge, logistics and resources to promote organic farming - to both
+      farmers and the public. As a valued member of our organization, you can access carefully researched and approved
+      content for reuse via this Content Portal.`,
+      de: `In unserem Verein geht es darum, Wissen, Logistik und Ressourcen zu teilen, um den ökologischen Landbau zu 
+      fördern - für beide Landwirte und die Öffentlichkeit. Als geschätztes Mitglied unserer Organisation können Sie auf 
+      sorgfältig recherchierte und genehmigte Daten zugreifen Inhalte zur Wiederverwendung über dieses Inhaltsportal.`,
+    },
+    Contact1: {
+      en: 'If you would like to contribute your ideas, please',
+      de: 'Wenn Sie Ihre Ideen einbringen möchten, bitte',
+    },
+    Contact2: {
+      en: 'contact us',
+      de: 'kontaktiere uns',
+    },
+  },
+  DemoInfoDialog: {
+    Info1: {
+      en: 'This is a slightly customized microsite of the Picturepark SDK - part of our online demo.',
+      de: 'Dies ist eine leicht angepasste Microsite des Picturepark SDK - Teil unserer Online-Demo.',
+    },
+    Info2: {
+      en: 'You can play around with your limited read-only access which simulates access of FOFA members.',
+      de: `Sie können mit Ihrem eingeschränkten schreibgeschützten Zugriff herumspielen, der den Zugriff 
+      von FOFA-Mitgliedern simuliert.`,
+    },
+    Info3: {
+      en: 'Explore other components of this demo.',
+      de: 'Entdecken Sie andere Komponenten dieser Demo.',
+    },
+  },
+  Items: {
+    DemoInfo: {
+      en: 'More info about this demo',
+      de: 'Weitere infos zu dieser demo',
+    },
+    CloseItem: {
+      en: 'Close item',
+      de: 'Element schließen',
+    },
+    Download: {
+      en: 'Download',
+      de: 'Download',
+    },
+    DownloadItem: {
+      en: 'Download item',
+      de: 'Download Artikel',
+    },
+    AddToBasket: {
+      en: 'Add to basket',
+      de: 'In Warenkorb legen',
+    },
+    RemoveFromBasket: {
+      en: 'Remove from basket',
+      de: 'Aus dem Warenkorb entfernen',
+    },
+    Filters: {
+      en: 'Filters',
+      de: 'Filter',
+    },
+    Basket: {
+      en: 'Basket',
+      de: 'Warenkorb',
+    },
+  },
+  Language: {
+    Selected: {
+      en: 'Selected',
+      de: 'Selektiert',
+    },
+  },
+  Profile: {
+    Login: {
+      en: 'Login',
+      de: 'Anmelden',
+    },
+    Logout: {
+      en: 'Logout',
+      de: 'Abmelden',
+    },
+    Profile: {
+      en: 'Profile',
+      de: 'Profil',
+    },
+    ManageAccount: {
+      en: 'Manage account',
+      de: 'Konto verwalten',
+    },
+    ContactAdmin: {
+      en: 'Contact admin',
+      de: 'Kontakt admin',
```

Comment:
Admin kontaktieren - I assume this is a link with a mailto where you can contact the administrator?

Reply:
Yes, it is a link with a mail too.
Done.